test/test_trezor.py

```diff
@@ -359,10 +357,11 @@ def trezor_test_suite(emulator, rpc, userpass, interface, model_t=False):
     parser.add_argument('emulator', help='Path to the Trezor emulator')
     parser.add_argument('bitcoind', help='Path to bitcoind binary')
     parser.add_argument('--interface', help='Which interface to send commands over', choices=['library', 'cli', 'bindist'], default='library')
-    parser.add_argument('--model_t', help='The emulator is for the Trezor T', action='store_true')
+    parser.add_argument('--model_1', help='The emulator is for the Trezor One', action='store_const', const='1', dest='model')
+    parser.add_argument('--model_t', help='The emulator is for the Trezor T', action='store_const', const='t', dest='model')
```

Comment:
Please use a [mutually exclusive group](https://docs.python.org/3/library/argparse.html#argument-groups).

```
group = parser.add_argument_group()
group.add_argument('--model_1', help='The emulator is for the Trezor One', action='store_const', const='1', dest='model')
group.add_argument('--model_t', help='The emulator is for the Trezor T', action='store_const', const='t', dest='model')
```

Reply:
Addressed in 267f2af7304354a90899c3cc517abf7fedc8891e